docs/scos/dev/feature-integration-guides/202307.0/glue-api/data-exchange-api/data-exchange-api-integration.md

```diff
@@ -136,6 +137,38 @@ class DynamicEntityBackendApiConfig extends SprykerDynamicEntityBackendApiConfig
 }
 ```
 
+4. The Data Exchange API comes with an exclusion list of database schemas that are not allowed to be configured. 
```

Comment:
Is this step optional?

Reply:
By default schema for Data Exchange API configurations is excluded and it's highly not recommended to get rid of it. But it is optional for schemas that customer prefers to exclude.